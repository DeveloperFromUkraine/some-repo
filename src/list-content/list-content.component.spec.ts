import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListContentComponent, ListContentCardComponent } from "./index";
import { ComponentTest } from "../../test/test-bed/component";
import { TranslationModule } from '../localization/translation.module';

describe('List Content', () => {
    let fixture: ComponentFixture<ListContentComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([TranslationModule], [ListContentComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListContentComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});