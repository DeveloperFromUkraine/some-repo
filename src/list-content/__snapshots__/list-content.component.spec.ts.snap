// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Content should match snapshot 1`] = `
<ign-list-content>
  <a
    class="skip-main"
    href="javascript:void(0)"
    id="skip-list"
    target="_self"
  >
    
  </a><div
    id="list-content-skip"
    role="main"
    tabindex="-1"
  />
</ign-list-content>
`;
